src/bindings/python/src/pyopenvino/core/compiled_model.cpp

```diff
@@ -40,13 +40,29 @@ void regclass_CompiledModel(py::module m) {
 
     cls.def("export_model", &ov::CompiledModel::export_model, py::arg("model_stream"));
 
+    cls.def(
+        "set_property",
+        [](ov::CompiledModel& self, const std::map<std::string, std::string>& properties) {
+            self.set_property({properties.begin(), properties.end()});
+        },
+        py::arg("properties"));
+
+    // todo: remove after Accuracy Checker migration to set/get_property API
     cls.def(
         "get_config",
         [](ov::CompiledModel& self, const std::string& name) -> py::object {
             return Common::from_ov_any(self.get_property(name)).as<py::object>();
         },
         py::arg("name"));
```

Comment:
Are we keeping it? I cannot see it in current header for CompiledModel class.

Reply:
If yes, same comment as other `from_ov_any` and simple types.

Reply:
We keep it till OMZ will migrate to the new API in order not to break the CI.

Reply:
I added deprecationWarning and 'todo comment' to remove it after OMZ migration